CMakeLists.txt

```diff
@@ -56,7 +57,12 @@ set( CMAKE_CXX_EXTENSIONS OFF )
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
 if( T8CODE_ENABLE_MPI )
-    find_package( MPI COMPONENTS C REQUIRED )
+    if( T8CODE_BUILD_FORTRAN_INTERFACE )
```

Comment:
why can we only build the fortran interface when MPI is enabled?

Reply:
As we discussed, for serial support the Fortran API has to be adapted accordingly. The workload of which is unclear. Besides, there is no use case right now for this. So limit to MPI only and add this feature later in case it is needed.